.github/workflows/linter-conan-v2.yml

```diff
@@ -1,124 +0,0 @@
-name: "[linter] Conan v2 migration"
-
-on:
-  pull_request:
-
-env:
-  PYTHONPATH: ${{github.workspace}}
-  PYVER: "3.8"
-  REQUIREMENTS: "pylint==2.14"
-
-jobs:
-  test_linter:
-    name: Test linter changes (v2 migration)
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Get changed files
-        uses: ./.github/actions/pr_changed_files
-        id: changed_files
-        with:
-          files: |
-            linter/**
-            .github/workflows/linter-conan-v2.yml
-
-      - name: Get Conan v1 version
-        id: parse_conan_v1_version
-        if: steps.changed_files.outputs.any_changed == 'true'
-        uses: mikefarah/yq@master
-        with:
-          cmd: yq '.conan.version' '.c3i/config_v1.yml'
-
-      - uses: actions/setup-python@v4
-        if: steps.changed_files.outputs.any_changed == 'true'
-        with:
-          python-version: ${{ env.PYVER }}
-
-      - name: Install requirements
-        if: steps.changed_files.outputs.any_changed == 'true'
-        run: |
-          pip install ${{ env.REQUIREMENTS }} conan==${{ steps.parse_conan_v1_version.outputs.result }}
-
-      - name: Execute linter over all recipes in the repository
-        id: linter_recipes
-        if: steps.changed_files.outputs.any_changed == 'true'
-        run: |
-          echo '## Linter summary (recipes)' >> $GITHUB_STEP_SUMMARY
-          pylint --rcfile=linter/pylintrc_recipe `ls recipes/*/*/conanfile.py | shuf -n 500` --output-format=json --output=recipes.json --score=y --exit-zero
```

Comment:
The `linter/pylintrc_recipe` is only used here and mentioned in docs/linters.md and docs/developing_recipes_locally.md. So it could be removed as well.